src/raw/cgroupv2.go

```diff
@@ -119,6 +120,9 @@ func (cg *CgroupsV2Fetcher) cpu(metric *cgroupstatsV2.Metrics) (CPU, error) {
 
 func (cg *CgroupsV2Fetcher) memory(metric *cgroupstatsV2.Metrics) (Memory, error) {
 	mem := Memory{}
+	b, _ := json.Marshal(metric)
+	log.Error("This is the payload of memory metrics" + string(b))
```

Comment:
debugging leftover ?

Reply:
Thanks for pointing it out, it was fixed locally with the other changes, but better to have a reminder! 😄